README.md

```diff
@@ -6,106 +6,235 @@
 [![Dependencies](https://david-dm.org/netlify/zip-it-and-ship-it/status.svg)](https://david-dm.org/netlify/zip-it-and-ship-it)
 [![Downloads](https://img.shields.io/npm/dm/@netlify/zip-it-and-ship-it.svg)](https://www.npmjs.com/package/@netlify/zip-it-and-ship-it)
 
-This module handles zipping up lambda functions with their dependencies before deployment. You are probably looking for
-[netlify-cli](https://github.com/netlify/cli) or [js-client](https://github.com/netlify/js-client).
+Creates Zip archives from Node.js or Go source files. Those archives are ready to be uploaded to AWS Lambda.
+
+This library is used under the hood by several Netlify features, including
+[production CI builds](https://github.com/netlify/build), [Netlify CLI](https://github.com/netlify/cli) and the
+[JavaScript client](https://github.com/netlify/js-client).
+
+Check Netlify documentation for:
+
+- [Netlify Functions](https://docs.netlify.com/functions/overview/)
+- [Bundling Functions in the CLI](https://www.netlify.com/docs/cli/#unbundled-javascript-function-deploys)
 
 ## Installation
 
 ```bash
 npm install @netlify/zip-it-and-ship-it
 ```
 
-## Usage
+## API
+
+### zipFunctions(srcFolder, destFolder, options?)
```

Comment:
`listFunctions` and `listFunctionsFiles` use h2 headers while `zipFunctions` and `zipFunction` use h3 headers.
We should try to be consistent I think.

Reply:
Right! Fixed.